CMakeLists.txt

```diff
@@ -464,8 +467,11 @@ endif()
 
 option(AVIF_BUILD_APPS "Build avif apps." OFF)
 option(AVIF_BUILD_TESTS "Build avif tests." OFF)
+option(AVIF_ENABLE_GTEST
+       "Build avif C++ tests, which depend on GoogleTest. Requires GoogleTest. Has no effect unless AVIF_BUILD_TESTS is ON." OFF
```

Comment:
Would you like to change the default to ON as you suggested before? If you do that, remember to delete the warning message at lines 515-516.